build/builtin_actors_gen.go

```diff
@@ -249,8 +249,8 @@ var EmbeddedBuiltinActorsMetadata []*BuiltinActorsMetadata = []*BuiltinActorsMet
 }, {
 	Network:      "calibrationnet",
 	Version:      13,
-	BundleGitTag: "v13.0.0-rc.3",
-	ManifestCid:  MustParseCid("bafy2bzacea4firkyvt2zzdwqjrws5pyeluaesh6uaid246tommayr4337xpmi"),
+	BundleGitTag: "v13.0.0",
+	ManifestCid:  MustParseCid("bafy2bzacect4ktyujrwp6mjlsitnpvuw2pbuppz6w52sfljyo4agjevzm75qs"),
```

Comment:
This doesn't seem correct, we shouldn't be updating the binary for calibnet in this PR (or else we won't sync that network anymore). We should update for everything _except_ calibnet by running `./pack.sh v13 v13.0.0 calibrationnet=v13.0.0-rc.3`.

Reply:
Pushed a fix in: https://github.com/filecoin-project/lotus/pull/11768/commits/f6e65bf0e5d2001c5d885f9b6b390eb3c5502861